lumino-explorer-api/src/main/java/org/rif/lumino/explorer/boot/configuration/WebConfiguration.java

```diff
@@ -10,7 +10,7 @@
 
   @Override
   public void addCorsMappings(final CorsRegistry registry) {
-    registry.addMapping("/**").allowedHeaders("*");
+    registry.addMapping("*").allowedHeaders("*");
```

Comment:
This change was neccesary because the old mapping was failing for urls like `/something/somethingelse/andsomethingelse`